packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -74,7 +76,7 @@ Panel.propTypes = {
    */
   isToggleable: PropTypes.bool,
   /**
-   * Function that is called when Panel is opened or closed.
+   * Function is triggered by componentDidUpdate or by toggleOpenState.
```

Comment:
internal information like componentDidUpdate is not relevant for a documentation.